src/tests/unit/conftest.py

```diff
@@ -35,6 +35,7 @@ def mock_config():
         # Load all defaults
         for key, value in yml['options'].items():
             cfg[key] = value['default']
+        cfg['enable-slack'] = True
```

Comment:
This patches the hookenv, I tend to think it should load defaults and tests should override settings that they want changed individually in each unit tests. I'm afraid this kind hides that you've changed the default config. This same dictionary is available on the helper instance in the test.